### DISCUSSION TITLE
Export to Excel Spreadsheet

### DISCUSSION THREAD

**Discussion by Angelofthenight091** [kwsch/PKHeX#4349] [Ideas]
Hey, I was wondering if it would be possible to have the ability implemented to export boxes, or entire pc of pokemon to an excel spreadsheet.   

For example 

Pokemon, Nickname, Nature, held item, ability, IVs Evs, Tera, attacks 1 - 4, OT, SID, Trainer Name.
I know myself I have a ton of Movesets across multiple Gens, and It would be pretty neat to be able to export all these to a spreadsheet to be able to cross reference.   However my idea is a niche, and unsure if anyone else has the same kind of obsession.

**Comment by kwsch** [comment]
From the main window, press ctrl-R to bring up the box data report. When you close the window, you can export all columns to CSV.

In the program settings, you can specify extra Properties (same as Batch Editor properties) to include in the report.

**Reply by Angelofthenight091** [reply]
Thanks, I was not aware this was a feature. Much appreciated
